src/domain/index.ts

```diff
@@ -61,7 +61,7 @@ export interface DomainServices {
  * @param appConfig - app config
  */
 export function init(repositories: Repositories, appConfig: AppConfig): DomainServices {
-  const noteService = new NoteService(repositories.noteRepository, repositories.noteRelationsRepository, repositories.noteVisitsRepository);
+  const noteService = new NoteService(repositories.noteRepository, repositories.noteRelationsRepository, repositories.noteVisitsRepository, repositories.editorToolsRepository);
```

Comment:
can you use shared methods instead of direct coupling of two editor tools and note services?

Reply:
there would be circular dependency (note service is already in shared, it can not use shared, in which it is required)